### PR TITLE
document-translation

### LINKED ISSUE #330: [DOC]54
**Provide a link to that doc page:**
Fixes #325 
**What is the defect and your suggestions on improvement:**
document translation  in docs/zh/samples/nonroot_access.md

### PR DESCRIPTION
**Pull request by njuwkf**
### Ⅰ. Describe what this PR does
I translated the content in docs/zh/samples/nonroot_access.md to  docs/en/samples/nonroot_access.md

### Ⅱ. Does this pull request fix one issue?

Fixes #330

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.

No tests needed.
### Ⅳ. Describe how to verify it

Just read the comment.
### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/en/samples/nonroot_access.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The above command creates a non-root user`fluid-user-1`
+
+**Create a directory that belongs to the user**
+```
+$ mkdir -p /mnt/nonroot/user1_data && \
+echo "This is fluid-user-1's data" > /mnt/nonroot/user1_data/data1 && \
+chown -R fluid-user-1:fluid-user-1 /mnt/nonroot/user1_data && \
+chmod -R 0750 /mnt/nonroot/user1_data
+```
+The above command creates a directory `user1_data` belonging to `fluid-user-1` in the `/mnt/nonroot` directory, We will use the `data1` file in the `user1_data` directory to simulate the data dedicated to `fluid-user-1`
```
Comment (TrafalgarZZZ) [inline docs/en/samples/nonroot_access.md:17]:
```suggestion
The above command creates a directory `user1_data` belonging to `fluid-user-1` in the `/mnt/nonroot` directory, We will use the `data1` file in the `user1_data` directory to represent the data owned by `fluid-user-1`
```
I think "represent" and  "owned by" may be better

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

File: docs/en/samples/nonroot_access.md (hunk 1 above)
Comment (njuwkf) [reply docs/en/samples/nonroot_access.md:17]: I think your suggestion is great，I will commit your suggestion.

**Review by TrafalgarZZZ — APPROVED** [review]
/lgtm

**Comment by xieydd** [comment]
Why this doc is not complete?

**Comment by njuwkf** [comment]
> Why this doc is not complete?

Because I just translated part of content，the rest will commit by others.

**Pull request merged**